app/backend/src/couchers/servicers/requests.py

```diff
@@ -74,7 +73,7 @@ def CreateHostRequest(self, request, context):
             if request.from_date > request.to_date:
                 context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.DATE_FROM_AFTER_TO)
 
-            if request.to_date < datetime.datetime.now().strftime("%Y-%m-%d"):
+            if request.to_date < now().strftime("%Y-%m-%d"):
```

Comment:
I would check whether the request dates are at or after the date in timezone UTC-12
Keeping into account the date at the location of the host does not add enough business value.

Reply:
I've fixed the comparison accordingly, please have another look.